File: VB/Q279518.Web/SuppliersReport.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.Xpo

Namespace Q279518.Web
	Partial Public Class SuppliersReport
		Inherits DevExpress.XtraReports.UI.XtraReport
		Public Sub New()
			InitializeComponent()
		End Sub
		Private Sub OnSuppliersCollectionResolveSession(ByVal sender As Object, ByVal e As ResolveSessionEventArgs) Handles suppliersCollection.ResolveSession
			e.Session = XpoHelper.GetNewSession()
		End Sub

	End Class
End Namespace

File: CS/Q279518.Web/SuppliersReport.cs
using System;
using DevExpress.Xpo;

namespace Q279518.Web {
    public partial class SuppliersReport :DevExpress.XtraReports.UI.XtraReport {
        public SuppliersReport() {
            InitializeComponent();
        }
        private void OnSuppliersCollectionResolveSession(object sender, ResolveSessionEventArgs e) {
            e.Session = XpoHelper.GetNewSession();
        }

    }
}
